assets/js/components/SpinnerButton.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * Site Kit by Google, Copyright 2022 Google LLC
```

Comment:
```suggestion
 * SpinnerButton component.
 *
 * Site Kit by Google, Copyright 2022 Google LLC
```

Reply:
This is not really needed since the `SpinnerButton component.` line has 0 value.